packages/x-components/tests/e2e/cucumber/next-queries/stubs/next-queries.stub.ts

```diff
@@ -0,0 +1,9 @@
+import { createNextQueryStub } from '../../../../../src/__stubs__/next-queries-stubs.factory';
+
+export const nextQueriesStub = {
```

Comment:
Add the `NextQueriesResponse` type

Reply:
And I don't know where to put this, but be careful with imports, for exmaple in `recommendations.spec` you are importing from a relative package, instead of node modules.

`import { TopRecommendationsRequest } from '../../../../../search-adapter/src/types/requests.types';`
should be
`import { TopRecommendationsRequest } from '@empathyco/x-adapter';`